virtual_ecosystem/models/abiotic/microclimate.py

```diff
@@ -183,7 +183,8 @@ def run_microclimate(
     )
 
     # Aerodynamic resistance soil, [s m-1]
-    aerodynamic_resistance_soil = data["aerodynamic_resistance_surface"].to_numpy()
+    # VIVI - this was ... = data["aerodynamic_resistance_surface"].to_numpy()
+    aerodynamic_resistance_soil = aerodynamic_resistance_understorey
```

Comment:
There is a difference between `surface` and `understorey` here, please leave as it is for now if possible, I am actively working on this code